aiogram/dispatcher/webhook.py

```diff
@@ -448,6 +448,17 @@ def _global_disable_web_page_preview():
             return bot.disable_web_page_preview
 
 
+class ProtectContentMixin:
+    def protect_content(self):
+        """
+        Protect content
+
+        :return:
+        """
+        setattr(self, "protect_content", True)
```

Comment:
Why default value is `True`?

Reply:
protect_content is verb describe action of protection of content. this method like https://github.com/aiogram/aiogram/blob/4d2d81138681d730270819579f22b3a0001c43a5/aiogram/dispatcher/webhook.py#L418 and https://github.com/aiogram/aiogram/blob/4d2d81138681d730270819579f22b3a0001c43a5/aiogram/dispatcher/webhook.py#L429 created for modify response by invoke method (is cleaner solution than change field of object)

it is not default value. This method change value of field.